src/index.html

```diff
@@ -12,7 +15,39 @@
       href="styles/index.scss"
```

Comment:
You are linking to a SCSS file in your HTML. However, browsers can't understand SCSS. You should compile your SCSS to CSS and link the CSS file in your HTML. Please update the href attribute to point to the compiled CSS file.